CMakeLists.txt

```diff
@@ -69,6 +69,8 @@ endif()
 # testing
 if(BUILD_TESTING)
```

Comment:
Please have a look at [How do I name test targets in CMake?](https://everest.github.io/nightly/general/faq.html#unit-tests) and adapt the condition in the if statement